src/h3lib/lib/faceijk.c

```diff
@@ -568,7 +576,7 @@ void _faceIjkPentToGeoBoundary(const FaceIJK* h, int res, GeoBoundary* g) {
                     edge0 = &v1;
                     edge1 = &v2;
                     break;
-                case KI:
+                // case KI:
```

Comment:
This is pure coverage mongering, same below.

Reply:
Please add a comment about why this is commented out

Reply:
I think I'll remove the comment, it doesn't seem to have improved coverage. There's some difference between my local `lcov` and what we're running in CI - this is marked as a missing branch locally, but commenting it out didn't improve CI coverage. I think the last `.03%` of coverage lost here is a result of changing the number of lines of code - would we be okay landing with a small loss in coverage?